src/passes/function_passes/CheckPointPass.cpp

```diff
@@ -237,6 +237,9 @@ struct CheckPointPass: public FunctionPass {
     }
     if (stackMaps.find(funName) == stackMaps.end() &&
         stackMaps.find(getTwinName(funName)) == stackMaps.end()) {
+      // This ensures `stackmap`/`patchpoint` calls which belong to the same
```

Comment:
I guess that should have been in the previous commit, but don't worry too much about changing it.

Reply:
Yes, I was too lazy to order the commits so I just amended the last one.